app/src/main/java/net/frju/flym/ui/about/AboutActivity.kt

```diff
@@ -22,13 +22,17 @@ import android.support.v7.app.AppCompatActivity
 import android.view.MenuItem
 import com.vansuita.materialabout.builder.AboutBuilder
 import net.fred.feedex.R
+import net.frju.flym.data.utils.PrefUtils
 
 
 class AboutActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        //Choose theme
+        setTheme(if (PrefUtils.getBoolean(PrefUtils.DARK_THEME, true)) R.style.AppTheme else R.style.AppThemeLight )
```

Comment:
It would be great but all activities do not use the same theme (e.g. R.style.AppTheme_NoActionBar for EntryDetailsActivity) so I would have to either "hardcode" the theme for each activity or leave it as a parameter, which is basically the same as it is now.